kDriveCore/Data/Api/Endpoint+Share.swift

```diff
@@ -52,7 +51,12 @@ public extension Endpoint {
 
     /// Share link file
     static func shareLinkFile(driveId: Int, linkUuid: String, fileId: Int) -> Endpoint {
-        Self.shareUrlV3.appending(path: "/\(driveId)/share/\(linkUuid)/files/\(fileId)")
+        shareUrlV3.appending(path: "/\(driveId)/share/\(linkUuid)/files/\(fileId)")
+    }
+
+    /// Some legacy calls like thumbnails require a V2 call
```

Comment:
The calls are not legacy. Versioning is done per endpoint. Some endpoints haven't changed and are /2/ and some are and are /3/
```suggestion
```